src/btree/bt_sync.c

```diff
@@ -207,13 +207,12 @@ __sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
 	if (WT_VERBOSE_ISSET(session, WT_VERB_CHECKPOINT)) {
 		WT_ERR(__wt_epoch(session, &end));
 		__wt_verbose(session, WT_VERB_CHECKPOINT,
-		    "__sync_file WT_SYNC_%s wrote:\n\t %" PRIu64
-		    " bytes, %" PRIu64 " pages of leaves\n\t %" PRIu64
-		    " bytes, %" PRIu64 " pages of internal\n\t"
-		    "Took: %" PRIu64 "ms",
+		    "__sync_file WT_SYNC_%s wrote: %" PRIu64
+		    " leaf pages (%" PRIu64 "B), %" PRIu64
+		    " internal pages (%" PRIu64 "B), and took %" PRIu64 "ms",
```

Comment:
This is the only one I'm not sure about.  It looks like it was intentionally formatted so that the information as visually clear and easy to spot.  I'm okay with the new format but I've not used this verbose much.